src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -3996,7 +3996,7 @@ static bool TakeOutParam(int y, out int x)
 True");
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/14214")]
```

Comment:
> [Fact(Skip = "https://github.com/dotnet/roslyn/issues/14214")] [](start = 7, length = 63)

Please do not disable the entire test. Instead, adjust it to use helpers that do not verify data flow and mark the adjustments with the reference to the issue. Also, please add WorkItem attribute.